src/epcpm/sunspectocsv.py

```diff
@@ -419,12 +431,81 @@ def gen(self) -> typing.List[typing.List[str], int]:
             row.uuid = self.wrapped.uuid
             row.class_name = "DataPointBitfield"
 
-        uses_interface_item = (
-            isinstance(parameter, epyqlib.pm.parametermodel.Parameter)
-            and parameter.uses_interface_item()
-        )
+        rows = []
+        points = list(self.wrapped.children)
+        for child in points:
+            builder = builders.wrap(
+                wrapped=child,
+                model_type=self.model_type,
+                parameter_uuid_finder=self.parameter_uuid_finder,
+                model_id=self.model_id,
+                model_offset=self.model_offset,
+                address_offset=self.address_offset,
+            )
+            child_row = builder.gen()
+            rows.append(child_row)
+
+        # Insert the parent DataPointBitfield ahead of the list of DataPointBitfieldMember rows.
+        rows.insert(0, row)
+
+        return rows, row.size
+
+
+@builders(epcpm.sunspecmodel.DataPointBitfieldMember)
+@attr.s
+class DataPointBitfieldMember:
+    wrapped = attr.ib(type=epcpm.sunspecmodel.DataPointBitfieldMember)
```

Comment:
Little class docstring? :)

Reply:
Added class docstrings for `sunspectocsv` in f0949f07b9962d8bb4d7117c1adfd0b4ccae0de9